src/test/ut/ut_cortxfs_xattr_ops.c

```diff
@@ -494,7 +499,7 @@ void listxattr_test(void **state)
  */
 int listxattr_test_teardown(void **state)
 {
-	int rc = 0;
+	int i, rc = 0, size;
```

Comment:
size name is misleading, should be count.

Reply:
fixed